.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
+ignore = D203,E226,E302,E41,F401,E402
+exclude =
+    .git,
+    __pycache__,
```

Comment:
We take care of this ```.gitignore```, so not needed here. Also remove the ignored ones, we'll be taking up all. You can completely remove .flake8 itself. Can you share the count of the warning you got while flake8?